app/models/event.rb

```diff
@@ -1,4 +1,9 @@
 class Event < ActiveRecord::Base
+  include Filterable
+
+  scope :event_type, -> (event_type) { where event_type_id: event_type}
```

Comment:
Use this with the parenthesis to keep consistency: `where(event_type_id: event_type)`; also; are you passing an id or the entire object to this scope? Maybe here we can change the param name to `event_type_id`, in this way we know what we're passing explicitly. What do you think?